src/index.ts

```diff
@@ -44,12 +45,90 @@ const localStorage = new EppoLocalStorage();
  */
 export class EppoJSClient extends EppoClient {
   public static instance: EppoJSClient = new EppoJSClient(localStorage);
+
+  public getAssignment(
+    subjectKey: string,
+    flagKey: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    subjectAttributes?: Record<string, any>,
+    assignmentHooks?: IAssignmentHooks,
+  ): string | null {
+    return super.getAssignment(subjectKey, flagKey, subjectAttributes, assignmentHooks, true);
```

Comment:
We're overriding the client's instance methods because the last argument (`obfuscated: boolean`) isn't meant to be available to the end user. It's determined by the SDK name and version